README.md

```diff
@@ -93,11 +94,60 @@ When passing through GPUs to the inner container, you may end up using associate
 
 Envbox will detect these mounts and pass them inside the inner container it creates, so that GPU-aware tools run inside the inner container can still utilize these libraries.
 
+Here's an example Docker command to run a GPU-enabled workload in Envbox. Note the following:
+
+1) The NVidia container runtime must be installed on the host (`--runtime=nvidia`).
+2) `CODER_ADD_GPU=true` must be set to enable GPU-specific functionality.
+3) When `CODER_ADD_GPU` is set, it is required to also set `CODER_USR_LIB_DIR` to a location where the relvant host directory has been mounted inside the outer container. In the below example, this is `/usr/lib/x86_64-linux-gnu` on the underlying host. It is mounted into the container under the path `/var/coder/usr/lib`. We then set `CODER_USR_LIB_DIR=/var/coder/usr/lib`. The actual location inside the container is not important **as long as it does not overwrite any pre-existing directories containing system libraries**.
+4) The location to mount the libraries in the inner container is determined by the distribution ID in the `/etc/os-release` of the inner container. If the automatically determined location is incorrect, you can set it manually via `CODER_INNER_USR_LIB_DIR`.
```

Comment:
> If the automatically determined location is incorrect

How will users know if this is the case?

Reply:
Their GPU libraries won't load. I don't think we need to explain this to them since it'll be quite obvious when nvidia-smi and whatever app they're running doesn't load

Reply:
Correct, I'll add a note about `nvidia-smi` complaining about libraries for context.